handlers/_swagger.js

```diff
@@ -252,12 +253,15 @@ function registerRoute(app, auth, additionalMiddleware, method, path, data, allo
             if (responseModelValidationLevel) {
                 var responseSender = res.send;
                 res.send = function (body) {
+                    var isJson = typeof body === 'string'; //body can come in as JSON or object, we want object
+                    body = isJson ? JSON.parse(body) : body;
                     var validationErrors, invalidBody;
+                    var validateResponse = validateResponseModels.bind({},res, body, data, logger, swaggerDoc);
```

Comment:
generally, I need an explanation of this addition and the change to the `responseSender` below

assuming this stays, I think we need a more specific name for this bound function, maybe `doValidateResponseModels` or `doResponseModelValidation` or `validateThisResponseModel` ...